fluentd/configs.d/openshift/es-ops-copy-config.conf

```diff
@@ -17,9 +17,9 @@
       # recreate/reload connections
       reload_connections false
       reload_on_failure false
-      flush_interval 5s
-      max_retry_wait 300
-      disable_retry_limit true
+      flush_interval "#{ENV['OPS_FLUSH_INTERVAL'] || ENV['ES_FLUSH_INTERVAL'] || '5s'}"
+      max_retry_wait "#{ENV['OPS_RETRY_WAIT'] || #{ENV['ES_RETRY_WAIT'] || '300'}"
+      disable_retry_limit "#{ENV['OPS_DISABLE_RETRY_LIMIT'] || #{ENV['ES_DISABLE_RETRY_LIMIT'] || 'true'}"
```

Comment:
What does this buy the user to set this to `false`?  As an admin, who would want to lose logs?